### DISCUSSION TITLE
Is there a way I can pre-download google fonts somewhere?

### DISCUSSION THREAD

**Discussion by bilal-fazlani** [squidfunk/mkdocs-material#5386] [Q&A] [answered]
I would like to run `mkdocs build` in an environment where it cant reach google.com
but when I enable cards, it tries to download fonts and fails the build.

Can I pre-download the font's somewhere so it does not try to download the files?

**Comment by squidfunk** [comment, marked as answer]
The built-in plugins save everything in the `.cache` folder, including the privacy plugin, the social plugin and the optimize plugin. You would just need to add those folders to your environment, specifically the contents of `.cache/plugin/social`.

**Reply by bilal-fazlani** [reply]
I will try this tonight

**Reply by bilal-fazlani** [reply]
it worked

**Reply by squidfunk** [reply]
Perfect, thanks for coming back and reporting!

**Reply by WraithLyd** [reply]
where 's `.cache`?

**Reply by squidfunk** [reply]
In the root folder of your project where `mkdocs.yml` resides.
